packages/vat-data/src/types.d.ts

```diff
@@ -23,24 +22,44 @@ type ActualBehavior<B> = {
   [Facet in keyof B]: FunctionsMinusContext<B[Facet]>;
 };
 
-type KindContext<S, B> = { state: S; facets: ActualBehavior<B> };
+type MultiKindContext<S, B> = { state: S; facets: ActualBehavior<B> };
 
-interface KindDefiner {
-  <P, S, B>(
+declare const kindHandleMarker: unique symbol;
+export type DurableKindHandle = { readonly [kindHandleMarker]: string };
+
+export type VatData = {
+  // virtual kinds
+  defineKind: <P, S, F>(
+    tag: string,
+    init: (...args: P) => S,
+    facet: F,
+    options?: {
+      finish?: (context: { state: S }, kind: FunctionsMinusContext<F>) => void;
```

Comment:
Not a type review, but a question on the API itself for @FUDCo. Any reason the `kind` is in is own arg, and we're not using the `self` of the `context`?